LICENSE.md

```diff
@@ -1,12 +1,7 @@
-<!---
-DO NOT EDIT !
-This file was generated automatically from 'src/generator/templates/LICENSE.md.mako'
-DO NOT EDIT !
--->
 The MIT License (MIT)
 =====================
 
-Copyright © `2015-2020` `Sebastian Thiel`
+Copyright 2015–2024 Sebastian Thiel
```

Comment:
Updating to the official template:

https://opensource.org/license/MIT

Reply:
9 years?? Oh my…!